front/app/utils/configs/participationMethodConfig/index.tsx

```diff
@@ -106,27 +109,25 @@ const ideationConfig: ParticipationMethodConfig = {
   supportsVotes: true,
   supportsComments: true,
   supportsTopicsCustomField: true,
-  onFormSubmission: (props: FormSubmissionMethodProps) => {
-    if (props.ideaId && props.idea) {
```

Comment:
Simplified / cleaned up the code a bit in this file by using destructuring and removing unnecessary conditons